pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_config_repo.adoc

```diff
@@ -0,0 +1,29 @@
+[id="configuring-a-repository-as-a-content-source-for-pantheon_{context}"]
+
+= Configuring a repository as a content source for Pantheon
+
+.Procedure
+
+. In a text editor, paste the content of the sample configuration file in xref:pantheon-yaml-file_{context}[] into a new text file. Name the file [filename]`pantheon2.yml`. This file is the Pantheon configuration file for your repository.
```

Comment:
I suggest: 
s / 'xref:pantheon-yaml-file_{context}[]' / 'xref:pantheon-yaml-file_{context}[pantheon-yaml-file]' or some such, so that the rendered text looks better.

Also, would it not be better to use 'link' instead of 'xref'?

Reply:
@theashiot Why is it better to use link instead of xref?

And why would the rendered text look better the way you suggest?